roles/install_kubectl_versions/tasks/process-install.yaml

```diff
@@ -14,26 +14,78 @@
   tags:
     - avinode_devenv_install_kubectl
 
-- name: Fetech kubectl checksum ({{ version }})
-  uri:
-    return_content: yes
-    url: https://dl.k8s.io/v{{ version }}/bin/{{ go_platform }}/{{ go_arch }}/kubectl.sha256
-  register: _kubectl_checksum
-  tags:
-    - avinode_devenv_install_kubectl
+- name: Install kubectl version
+  block:
+  - name: Create temp file for checksum
```

Comment:
Indent?  Tabbing is inconsistent.

Reply:
I can only se spaces and no tab characters. What is the inconsistency?